### PR TITLE
Driver's License Scan Functionality

### PR DESCRIPTION
**Pull request by sam-warren**

### CODE REVIEW

File: roadside-forms-frontend/frontend_web_app/src/components/Forms/VehicleImpoundment/vehicleImpoundmentReason.js [hunk 1]
```diff
@@ -1,28 +1,86 @@
...
-}
+  return (
+    <div className="border-design-form left text-font">
+      <h3>Impoundment for Driving Behaviour</h3>
+      <Row>
+        <Col className="left checkboxs">
+          <Checkbox name="excessive_speed">
+            Excessive Speed
+            <span className="light-text">
+              {" "}
```
Comment (JustinMacaulay) [inline roadside-forms-frontend/frontend_web_app/src/components/Forms/VehicleImpoundment/vehicleImpoundmentReason.js:16]: Not sure what this is for...

**Review by JustinMacaulay — COMMENTED** [review]
Just a comment on the weird {" "}  things being added by the formatter

File: roadside-forms-frontend/frontend_web_app/src/components/Forms/VehicleImpoundment/vehicleImpoundmentReason.js (hunk 1 above)
Comment (sam-warren) [reply roadside-forms-frontend/frontend_web_app/src/components/Forms/VehicleImpoundment/vehicleImpoundmentReason.js]: Removed.

**Review by JustinMacaulay — APPROVED** [review]
Looks good

**Pull request merged**